lib/data_hygiene/bulk_organisation_updater.rb

```diff
@@ -0,0 +1,67 @@
+require "csv"
+
+module DataHygiene
+  class BulkOrganisationUpdater
+    def initialize(filename)
+      @filename = filename
+    end
+
+    def call
+      CSV.foreach(filename, liberal_parsing: true, headers: :first_row) do |row|
+        process_row(row)
+      end
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+  private
+
+    attr_reader :filename
+
+    def process_row(row)
+      document = find_document(row)
+      return unless document
+
+      organisations = find_organisations(row)
+      update_document(document, organisations)
+    end
+
+    def find_document(row)
+      path = row.fetch("Path").to_s
+      edition = Edition.find_by(base_path: path)
+
+      if edition.nil?
+        puts "error: #{path}: could not find edition"
+      else
+        document_content_id = edition.document.content_id
+      end
+      document_content_id
+    end
+
+    def find_organisations(row)
+      all_organisations = Queries::GetLinkables.new(document_type: "organisation").call
+      organisations = row.fetch("All organisations")
+      return [] unless organisations
+
+      new_organisations = []
+      organisations.split(",").map do |slug|
+        organisation = all_organisations.detect { |org| org.base_path == "/government/organisations/#{slug}" }
```

Comment:
I think this code might struggle with rogue whitespace.

Given an example row:

| Path | All organisations |
| --- | --- |
| the-warm-home-discount-scheme | department-for-energy-security-and-net-zero, department-for-business-energy-and-industrial-strategy |

There are spaces in between each of the slugs.

When we split them by `","`:

```ruby
"dept-one, dept-two, dept-three".split(",") # => ["dept-one", " dept-two", " dept-three"]
```

Note the `space` character at the beginning of the second and third slugs.

I think this will result in problems when trying to match the organisation by `base_path`:

```ruby
"/government/organisations/#{slug}" # => "/government/organisations/ dept-two"
```